extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -201,6 +207,28 @@ class AmpStickyAd extends AMP.BaseElement {
       this.viewport_.updatePaddingBottom(0);
     });
   }
+
+  /**
+   * To check for background-color alpha and force it to be 1.
+   * Whoever call this needs to make sure it's in a vsync.
+   * @private
+   */
+  forceOpacity_() {
+    if (!isExperimentOn(this.win, TAG)) {
+      return;
+    }
+    const background = this.win.getComputedStyle(this.element)
+        .getPropertyValue('background-color');
+    if (!startsWith(background, 'rgba')) {
+      return;
+    }
+    user().warn('AMP-STICKY-AD', 'Do not allow container to be transparent');
```

Comment:
I found that when rgba has an alpha value equal or greater than one. `getComputedStyle` background-color will always return `rgb` instead. I tested this with Chrome and Safari, haven't tested with hsla, but I assume it would be the same.